### PR TITLE
table parsing and key value parsing samples

### PR DESCRIPTION
**Pull request by dizcology**
two samples:
- table parsing
- key value parsing

### CODE REVIEW

File: document/cloud-client/parse_key_value.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google LLC. All Rights Reserved.
```
Comment (nnegrey) [inline document/cloud-client/parse_key_value.py]: License should just be: `# Copyright 2019 Google LLC`

Without the `.` period after LLC
And drop the `All Rights Reserved.`

File: document/cloud-client/parse_key_value.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+
+
+# [START document_parse_key_value]
```
Comment (nnegrey) [inline document/cloud-client/parse_key_value.py]: Should we put the start just above the `import` lines and below the function comment?

Then we also started adding the defaults for the parameters as commented out variables?

File: document/cloud-client/parse_key_value.py (hunk 1 above)
Comment (dizcology) [reply document/cloud-client/parse_key_value.py]: done.

File: document/cloud-client/parse_key_value.py (hunk 2 above)
Comment (dizcology) [reply document/cloud-client/parse_key_value.py]: updated, please take a look.

**Comment by dizcology** [comment]
replaced by https://github.com/GoogleCloudPlatform/python-docs-samples/pull/2551

**Pull request closed without merging**